docs/_templates/index.html

```diff
@@ -0,0 +1,302 @@
+                    <div class="typewriter_container">
+                        <h1 class="lead typewriter"><b>The endgame Python stack debugger</b></h1>
+                    </div>
+                    <p class="lead text-muted">
+                        PyStack reports the stack frames of a running Python process or a Python core dump
+                    </p>
+                    <p>
+                        <a href=" {{ pathto('overview') }} "
+                            class="btn btn-primary filled-pystack btn-lg">Documentation</a>
+                        <a href="https://github.com/bloomberg/pystack/discussions"
+                            class="btn btn-secondary btn-lg">Community</a>
+                    </p>
+                    <div class="mouse"> <span></span> </div>
+                </div>
+            </div>
+        </section>
+
+        <!-- Marketing messaging and featurettes
+      ================================================== -->
+        <!-- Wrap the rest of the page in another container to center all the content. -->
+
+        <div class="container marketing">
+
+            <hr class="featurette-divider">
+
+            <!-- Three columns of text below the carousel -->
+            <div class="row">
+                <div class="col-lg-4">
+                    <img class="rounded-circle rounded-header-image"
+                        src="{{ pathto('_static/images/box.png', 1) }}"
+                        alt="Cardboard box" width="140" height="140">
+                    <h2>Processes and core files</h2>
+                    <p>
+                        No matter the situation, PyStack is your reliable
+                        companion for program analysis. If a process is running
+                        more slowly than you expected, PyStack can tell you what
+                        it's doing. If it's hanging, PyStack can tell you where
+                        it got stuck. If it crashed, PyStack can even tell you
+                        what it was doing from the core dump it left behind.
+                    </p>
+                    <p><a class="btn btn-secondary filled-pystack"
+                        href="{{ pathto('process') }}"
+                        role="button">Get started &raquo;</a></p>
+                </div><!-- /.col-lg-4 -->
+                <div class="col-lg-4">
+                    <img class="rounded-circle rounded-header-image"
+                        src="{{ pathto('_static/images/python.png', 1) }}"
+                        alt="Python logo" width="140" height="140">
+                    <h2>Ready for anything</h2>
+                    <p>
+                        PyStack thrives in extreme conditions, always ensuring
+                        accurate insights. It can handle deep call stacks like
+                        a breeze. It can get you an comprehensive report even if
```

Comment:
```suggestion
                        a breeze. It can get you a comprehensive report even if
```

Alternatively, it could read "It can provide a comprehensive report even if..."

Reply:
I prefer "get you a" over "provide a" just to be a bit less formal (my writing tends to be overly formal, and part of my editing pass is always to try to sound at least a bit more folksy, haha)